src/pages/TableList.js

```diff
@@ -52,8 +58,20 @@ export default class TableList extends Component {
   fetch2ptShootersData() {
     providers.get2ptShooters()
       .then(data => {
-        this.setState(() => ({
-          content: [] //content: JSON.stringify(data.shooters_2pt)
+          let dataShooters_2pt = data.shooters_2pt;
+            dataShooters_2pt=dataShooters_2pt.map(x=> 
+              Object.values({ standing: x.standing,
```

Comment:
Why are you creating new object inside object.values call?  Thats pretty much a nonsense, you create new object and then extracts its values into array and throws away the newly created object.
Please create an array and list columns in it ...

[ shooter.standing, shooter.player.display_name, shooter.team.name,  ... ]